src/routes/admins.js

```diff
@@ -174,12 +173,12 @@ export default (apiServer) => {
     }
 
     const result = await s3.upload(uploadParams).promise()
-    await patchOne(AdminModel, { id: req.params.id }, { profilePicture: baseUrl + result.Key })
+    await patchOne(AdminModel, { id: req.params.id }, { profilePicture: result.Key })
     return {
       status: 200,
       result: {
         success: true,
-        profilePicture: baseUrl + result.Key
+        profilePicture: result.Key
```

Comment:
We discussed that the name of this property should be profilePicturePath, since it does not contain the full url. Please update it in the model as well.